### ISSUE TITLE
Keyboard focus order is not logical as it moves to 'ABC' table control instead of checkbox.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7234]
**Test Environment**
OS: Windows_11
Version: 21H2
OS Build: 22000.795
Browser: Version 103.0.1264.71 (Official Build) (64-bit)
Tool: AI for web
URL: https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/row-selection

**Repro Steps:**
1. Open [Angular Grid Row Selection | Ignite UI for Angular | infragistics](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/row-selection)
2. Grids excel style filtering pen page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Verify the issue that focus order is proper on table content.

**Actual Behavior:**
Keyboard focus order is not logical as it moves to 'ABC' table control instead of checkbox.

**Expected Behavior**
Keyboard focus order should be logical. First the focus should move to the checkbox then to 'ABC' table control.
<img width="958" alt="MAS2 4 3_keyboard focus order is incorrect" src="https://user-images.githubusercontent.com/93735775/183064761-39df1a94-593a-4a4b-8cb4-e705ef23138d.png">

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
